pages/1.11/security/oss/haproxy-adminrouter/index.md

```diff
@@ -1,24 +1,26 @@
 ---
 layout: layout.pug
 excerpt: Configuring HAProxy in front of an Admin Router
-title: Configuring HAProxy in Front of Admin Router
-navigationTitle: HAProxy in Front of Admin Router
-menuWeight: 6
+title: HAProxy and Admin Router
+navigationTitle: HAProxy and Admin Router
+menuWeight: 30
 ---
 
 <!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
 
-You can use HAProxy to set up an HTTP proxy in front of the DC/OS [Admin Router](/1.11/overview/architecture/components/#admin-router). For example, this can be useful if you want to present a custom server certificate to user agents connecting to the cluster via HTTPS. DC/OS does not currently support adding your own certificates directly into Admin Router.
+You can set up secure HTTPS communication using a custom server certificate with your DC/OS cluster by [setting up a proxy](#HAProxy) between the Admin Router and user agent requests coming in from outside of the cluster. The HTTP Proxy must perform on-the-fly HTTP request and response header modification, because DC/OS is not aware of the custom hostname and port that is being used by user agents to address the HTTP proxy.
```

Comment:
I know this wasn't reflected here before, but we would like to say "Admin Router" instead of "the Admin Router" throughout all of documentation, architecture diagrams, and developer notes.